pages/about.html

```diff
@@ -1,12 +1,33 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
-</head>
-<body>
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About Me</title>
+    <link rel="stylesheet" href="../styles/styles.css">
+  </head>
   
-</body>
+  <body class="wallpaper">
+    <header>
+      <nav class="page-links">
+        <a href="/Users/nancylee/Developer/projects/personal-portfolio-site/pages/index.html">Home</a>
```

Comment:
Careful with your links! This link says to look specifically for the file on your computer, which no one else has access to. Instead, you want to do relative paths, similar to what you did for the CSS file linking.
Here is a great resource on absolute vs relative paths:  https://www.geeksforgeeks.org/absolute-relative-pathnames-unix/

This applies to the image links as well.